core/src/main/scala/com/mesosphere/usi/core/protos/ProtoBuilders.scala

```diff
@@ -91,7 +91,12 @@ private[usi] object ProtoBuilders {
 
     reservations.foreach(b.addReservations)
 
-    if (allocationInfo != null) b.setAllocationInfo(allocationInfo)
+    if (allocationInfo != null) {
```

Comment:
`allocationInfo` is also used in `newOffer` method in the same class, may want to change there as well.